sdk/textanalytics/Azure.AI.TextAnalytics/perf/Azure.AI.TextAnalytics.Perf/Scenarios/RecognizeHealthcareEntities.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Test.Perf;
+
+namespace Azure.AI.TextAnalytics.Perf
+{
+    public sealed class RecognizeHealthcareEntities: TextAnalyticsTest<PerfOptions>
```

Comment:
I know in TA we decided to test something else and not healthcare because healthcare is an LRO and we don't have a good perf testing story for LROs.
I don't remember which scenario we decided to test, but I added the v-team so they can help